app/server/lib/updateChecker.ts

```diff
@@ -33,12 +34,23 @@ export async function checkForUpdates(gristServer: GristServer) {
   return await response.json();
 }
 
-export async function updateGristServerLatestVersion(gristServer: GristServer) {
+export async function updateGristServerLatestVersion(
+  gristServer: GristServer,
+  doItAnyway = false,
```

Comment:
I think there's a small tendency in codebase to name this kind of variable force*, like forceCheck.